Updated readme and added steps for setting up theme (#30)
mitodl/mitxpro-theme: README.md

For detail about themes, please see [edx-docs about themes](https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/changing_appearance/theming/overview_themes.html).

> **Note:** Please use `koa` branch from the theme repository if you are intergating it with `koa` release of `Open edX`.

Reviewer: ```suggestion
**Note:** Please use `koa` branch from the theme repository if you are integrating it with `koa` release of `Open edX`.
```

Make sure you have MIT xPRO base url set in your settings. You can either add it in `env` files or simply add it in `edx-platform/lms/envs/private.py`. e.g

`XPRO_BASE_URL="http://xpro.odl.local:8053/"`

> For each Open edX component that you want to apply a theme to, add the absolute path of the themes directory to the `COMPREHENSIVE_THEME_DIRS` configuration property.

Reviewer: ```suggestion
To apply themes in Open edX you have to add the absolute path of the themes directory to the `COMPREHENSIVE_THEME_DIRS`  in `lms.yml` and `studio.yml`.
```
